Sources/Orbit/Components/CountryFlag.swift

```diff
@@ -8,71 +8,58 @@ import SwiftUI
 /// - Note: [Orbit definition](https://orbit.kiwi/components/countryflag/)
 public struct CountryFlag: View {
 
+    public static let padding: CGFloat = 1
+    
     let countryCode: String
-    let height: Height
+    let size: Icon.Size
     let border: Border
 
     public var body: some View {
         SwiftUI.Image(countryCode.lowercased(), bundle: .current)
             .resizable()
             .scaledToFit()
-            .frame(height: height.value)
             .clipShape(clipShape)
             .overlay(
-                clipShape.stroke(border.color, lineWidth: BorderWidth.thin)
+                clipShape.strokeBorder(border.color, lineWidth: BorderWidth.hairline)
+                    .blendMode(.darken)
             )
+            .padding(Self.padding)
+            .frame(width: size.value)
+            .fixedSize()
     }
 
-    @ViewBuilder var clipShape: some Shape {
-        RoundedRectangle(cornerRadius: height.value / 8)
+    var clipShape: some InsettableShape {
+        switch border {
+            case .none:
+                return RoundedRectangle(cornerRadius: 0)
+            case .default:
+                return RoundedRectangle(cornerRadius: size.value / 10)
+        }
     }
 }
 
 // MARK: - Inits
 public extension CountryFlag {
 
     /// Creates Orbit CountryFlag component.
-    init(_ countryCode: String, height: Height = .default, border: Border = .default) {
+    init(_ countryCode: String, size: Icon.Size = .normal, border: Border = .default) {
```

Comment:
Not sure how long this has been here, but don't we want an enum for the countries?